test/e2e-cypress/static/pages/json-schema-2020-12-default-model-expand-depth/json-schema-2020-12-default-model-expand-depth.yaml

```diff
@@ -15,6 +15,11 @@ paths:
           content:
             application/json:
               schema:
+                $vocabulary: 
```

Comment:
I was not sure about the usage of `$vocabulary`, let me know if this one is fine here.

Reply:
In JSON Schema 2020-12, $vocabulary keyword has following shape:

```
  "$vocabulary": {
    "https://json-schema.org/draft/2020-12/vocab/core": true,
    "https://json-schema.org/draft/2020-12/vocab/applicator": true,
    "https://json-schema.org/draft/2020-12/vocab/validation": true,
    "https://json-schema.org/draft/2020-12/vocab/meta-data": true,
    "https://json-schema.org/draft/2020-12/vocab/format-annotation": false,
    "https://example.com/my-custom-vocab": true
  },
```

The above is just an example, but you can use  it in your fixture.